Telegram/server.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * TODO(developer): 
+ * Add your service key to the current folder.
+ * Uncomment and fill in these variables.
+ */
+// const projectId = 'my-project';
+// const location = 'global';
+// const agentId = 'my-agent';
+// const languageCode = 'en'
+// const TELEGRAM_TOKEN='1234567898:ABCdfghTtaD8dfghdfgh45sdf65467M';
+// const SERVER_URL='https://example.com';
+const getUuid = require('uuid-by-string');
```

Comment:
Please consider using this package: https://www.npmjs.com/package/uuid

Looks like it is more popular and has better support.